jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/ToolkitAddConnectionDialog.kt

```diff
@@ -45,6 +45,7 @@ data class ConnectionDialogCustomizer(
     val header: String? = null,
     val helpId: HelpIds? = null,
     val replaceIamComment: String? = null,
+    val customizerId: String? = null
```

Comment:
might be cleaner to just pass in the list of scopes we're requesting, defaulting to `sso:account:access`